model/riscv_sys_regs.sail

```diff
@@ -750,13 +752,40 @@ function read_seed_csr() -> xlenbits = {
 /* Writes to the seed CSR are ignored */
 function write_seed_csr () -> option(xlenbits) = None()
 
+/*  mseccfg is a XLEN-bit read/write register that controls */
+/*  security features */
+bitfield MSeccfg : bits(64) = {
+  // Reserved WPRI bits.
+  wpri_2 : 63 .. 34,
+  // PMM bits of Pointer Masking Extension
+  PMM    : 33 .. 32,
```

Comment:
Please can you add an enum and mapping for PMM to make the code more readable/robust? I.e.

```
enum PointerMaskingMode {
  Disabled,
  Reserved,
  Enabled7,
  Enabled16,
}

mapping pmm_bits : PointerMaskingMode <-> bits(2) = {
  Disabled <-> 0b00,
  Reserved <-> 0b01,
  Enabled7 <-> 0b10,
  Enabled16 <-> 0b11,
}
```

And then replace all literals with `pmm_bits(Enabled7)` or whatever.